pages/1.10/installing/oss/cloud/gce/index.md

```diff
@@ -6,283 +6,238 @@ navigationTitle: GCE
 menuWeight: 3
 ---
 
-You can configure a DC/OS cluster on Google Compute Engine (GCE) by using these scripts.
+The recommended way for deploying a OSS DC/OS cluster on GCE is using [Terraform](#terraform).
 
-- [Configure bootstrap node](#bootstrap)
-- [Install the DC/OS GCE scripts](#install)
-- [Configure parameters](#configure)
 
-**Important:** Upgrades are not supported with this installation method.
+# Terraform
+ **Disclaimer: Please note this is a [community driven project](https://github.com/dcos/terraform-dcos/tree/master/gcp) and not officially supported by Mesosphere directly.**
 
 ## Prerequisites
-- Google Cloud Console account
+- [Terraform 0.11.x](https://www.terraform.io/downloads.html)
+- GCP Cloud Credentials. _[configure via: `gcloud auth login`](https://cloud.google.com/sdk/downloads)_
+- SSH Keys
+- Existing Google Project. Soon automated with Terraform using project creation as documented [here.](https://cloud.google.com/community/tutorials/managing-gcp-projects-with-terraform)
+
+## Install Google SDK
+
+Run this command to authenticate to the Google Provider. This will bring down your keys locally on the machine for terraform to use.
+
+```bash
+$ gcloud auth login
+$ gcloud auth application-default login
+```
+
+## Configure your GCP SSH Keys
+
+Set the private key that you will be using to your ssh-agent and set public key in terraform. This will allow you to log into to the cluster after DC/OS is deployed and also helps Terraform setup your cluster at deployment time.
+
+```bash
+$ ssh-add ~/.ssh/your_private_key.pem
+```
+
+```bash
+$ cat desired_cluster_profile.tfvars
+gcp_ssh_pub_key_file = "INSERT_PUBLIC_KEY_PATH_HERE"
+...
+```
+
+## Configure a Pre-existing Google Project
+
+Currently terraform-dcos assumes a project already exist in GCP to start deploying your resources against. This repo soon will have support for terraform to create projects on behalf of the user soon via this [document](https://cloud.google.com/community/tutorials/managing-gcp-projects-with-terraform). For the time being a user will have to create this project before time or leverage an existing project.
```

Comment:
Change: 
"This repo soon will have support for terraform to create projects on behalf of the user soon via this.."
to:
"This repo will soon have support for terraform to create projects on behalf of the user via this..."

Reply:
Addressed